moveit_ros/planning/rdf_loader/src/synchronized_string_parameter.cpp

```diff
@@ -123,7 +123,7 @@ bool SynchronizedStringParameter::waitForMessage(const rclcpp::Duration& timeout
   auto const nd_name = std::string(node_->get_name()).append("_ssp_").append(name_);
   auto const temp_node = std::make_shared<rclcpp::Node>(nd_name);
   string_subscriber_ = temp_node->create_subscription<std_msgs::msg::String>(
-      name_, rclcpp::QoS(1).transient_local().reliable(),
+      name_, rclcpp::QoS(1).transient_local().reliable(),  // TODO: ?
```

Comment:
@tylerjw @henningkayser @JafarAbdi should this be changed to `rclcpp::SystemDefaultsQoS()` too or can you help us with a good comment that explains this exception?

Reply:
this should not be changed. transient_local takes care of supporting late subscriptions

Reply:
@Shobuj-Paul Can you replace the `// TODO: ?`  with something like `// "transient_local" is required for supporting late subscriptions.`

Reply:
Sure, I'll make the change.